index.php

```diff
@@ -307,7 +307,7 @@
                         <?php include('images/pantheon/actions/system-search.svg'); ?>
                         Applications
                     </div>
-                    <div id="slingshot-arrow"><img src='images/slingshot/arrow.svg'></div>
+                    <div id="slingshot-arrow"><img src='images/slingshot/arrow.svg' alt=""></div>
```

Comment:
Remove the alt completely if you are not setting text (which you shouldn't here)

Reply:
This is not correct, many screen readers will read the URL if no alt tag is set. Decorative images should have empty alt tags.